docs/ui/interactions/fragments/web/installation.md

```diff
@@ -0,0 +1,26 @@
+## Installation
+
+<docs-filter framework="react">
+
+```
+yarn add aws-amplify @aws-amplify/ui-react
```

Comment:
If we are looking to use this on the `preview-tag` we will probably want to have this point to the specific version to place this into.

Reply:
Yep, I agree^

Reply:
On it